src/services/twilio-api/get-help-doc-link.js

```diff
@@ -0,0 +1,64 @@
+const jsonMap = require('../api-doc-mapping.json');
+// json structure = url: {path : {topics: {}, commands: {}}, subpath : {wherever applicable}}
+const urlMap = new Map(Object.entries(jsonMap));
+
+const getSubroute = (cmdDetailsArr, subpathMap) => {
+  const subdomain = cmdDetailsArr[3];
+  // apiName/version to locate sub url in json map
+  const apiVersion = `${cmdDetailsArr[1]}/${cmdDetailsArr[2]}`;
+  const subUrlArr = subpathMap.get(apiVersion);
+  if (subUrlArr === undefined) {
+    return '';
+  }
+  const subUrlMap = new Map(Object.entries(subUrlArr));
+  const subUrl = subUrlMap.get(subdomain);
+  return subUrl;
+};
+
+const getRootPath = (cmdDetailsArr, subpathMap, topicsMap, commandsMap) => {
+  const topicName = cmdDetailsArr[0];
+  const apiName = cmdDetailsArr[1];
+  let url = '';
+
+  if (topicName === 'api') {
+    // get the array of apis and convert to a map
+    const arrOfAllApis = topicsMap.get(topicName);
+    if (!arrOfAllApis) {
+      return '';
```

Comment:
Can we cover this case in a UT?

Reply:
Since the `arrOfAllApis` will always have a value, because of the static json, this is not covered in a UT.

Reply:
Can we not give a temp json while testing, so this check gets covered?

Reply:
I couldn't find a pretty way to mock the json. Did this [here] for covering it.(https://github.com/twilio/twilio-cli/blob/44ccef891dffd9501e9950eeeb143dcd093e7fad/test/services/twilio-api.test.js#L189)